default.yaml

```diff
@@ -130,6 +130,19 @@ rateLimits:
 
 accounts:
   initialStatus: active
+  # customFields:
+  #   - name: "customField"
+  #     # default: "string"
+  #     type: <"string" | "integer" | "float" | "boolean"> # optional
+  #     defaultValue: <value> # optional
+  #     # default: false - true if field is an array of type
+  #     array: <true | false> # optional
+  #     # default: false - true if field can be updated by the user
+  #     editable: <true | false> # optional
+  #     # default: false - true if field is required by mutations
+  #     required: <true | false> # optional
+  #     # default: false - true if field should have an index in db
+  #     index: <true | false> # optional
```

Comment:
no default for this one?

Reply:
updated in docs https://github.com/GaloyMoney/galoy/pull/1465/files#diff-3233f46d946532b99e17ec18cdf28ed0917b52aadcb201563cbd167491197709R14